liana-gui/src/installer/message.rs

```diff
@@ -81,6 +81,7 @@ pub enum DefineBitcoind {
 #[derive(Debug, Clone)]
 pub enum DefineElectrum {
     ConfigFieldEdited(electrum::ConfigField, String),
+    ValidDomainChanged(bool),
```

Comment:
As `validate_domain` is part of the config, I feel that this should somehow also be part of `ConfigFieldEdited`. Perhaps you can use something like `ConfigFieldEdited(electrum::ConfigFieldValue)` where `ConfigFieldValue` is:
```
pub enum ConfigFieldValue {
    Address(String),
    ValidateDomain(bool),
}
```

Reply:
`ValidDomainChanged(true)` is shorter than
`ConfigFieldEdited(electrum::ConfigFieldValue(electrum::ConfigFieldValue::ValidateDomain(bool)))`
the event system of iced is already overwhelming imho, we should avoid "overwrapping" it only make the codebase less readable & more error prone

Reply:
Wouldn't it be just `ConfigFieldEdited(electrum::ConfigFieldValue::ValidateDomain(bool))`?

And by removing `electrum::` it would be just `ConfigFieldEdited(ConfigFieldValue::ValidateDomain(bool))`.

It doesn't seem too bad, but we can leave it for now. If we have a separate message per field, then we should at some point change `ConfigFieldEdited` to `AddressEdited`.

Reply:
at this point of the release cycle i'd like to bring this PR quickly to a mergeable state or put it on hold: to avoid everyone testing under the christmas tree :smile:

Reply:
That's OK, I meant it as something to consider in the future.